### PR TITLE
Fetch notification when load setting page

### PR DESCRIPTION
**Pull request by xjiang-at-wiris**
## Description

Fix error for MathType for TinyMCE (legacy) warning, that can be displayed multiple times on the Site administration.

## Steps to reproduce
(If this is your first time running Moodle, please following this instruction to configure: https://github.com/wiris/wiris-moodle-docker/blob/main/README.md)

1. Use Moodle > 4.1.
2. MathType for TinyMCE (legacy) is installed
3. Log in Moodle as an admin
4. Go to Site Administration
5. Go to Plugins > Filter > MathType by WIRIS
6. Change Render Type from Client to PHP.
7. Check that the warning only shows one time

---

[taskid 44423](https://wiris.kanbanize.com/ctrl_board/2/cards/44423/details/)

### CODE REVIEW

File: settings.php [hunk 1]
```diff
@@ -29,6 +29,8 @@
     global $CFG;
     global $wirisconfigurationclass;
 
+    \core\notification::fetch();
```
Comment (jgonzalez-at-wiris) [inline settings.php:32]: Can you please add a comment explaining why is this line here?

**Review by jgonzalez-at-wiris — CHANGES_REQUESTED** [review]

File: settings.php (hunk 1 above)
Comment (xjiang-at-wiris) [reply settings.php]: updated!

**Review by carla-at-wiris — CHANGES_REQUESTED** [review]
The following error appears when trying to change the Filter configuration:

```
Error output, so disabling automatic redirect.

Script /admin/settings.php?section=filtersettingwiris mutated the session after it was closed: $SESSION->cachestore_session: default_session-core/coursecat,default_session-core/courseeditorstate,default_session-core/navigation_expandcourse $SESSION->navcache
line 793 of /lib/classes/session/manager.php: call to debugging()
line 162 of /lib/classes/shutdown_manager.php: call to core\session\manager::check_mutated_closed_session()
line ? of unknownfile: call to core_shutdown_manager::shutdown_handler()
```

It stays then on an infinite loop, not letting me apply any change to the configuration.
The warning does not appear either.

**Comment by xjiang-at-wiris** [comment]
> The following error appears when trying to change the Filter configuration:
> 
> ```
> Error output, so disabling automatic redirect.
> 
> Script /admin/settings.php?section=filtersettingwiris mutated the session after it was closed: $SESSION->cachestore_session: default_session-core/coursecat,default_session-core/courseeditorstate,default_session-core/navigation_expandcourse $SESSION->navcache
> line 793 of /lib/classes/session/manager.php: call to debugging()
> line 162 of /lib/classes/shutdown_manager.php: call to core\session\manager::check_mutated_closed_session()
> line ? of unknownfile: call to core_shutdown_manager::shutdown_handler()
> ```
> 
> It stays then on an infinite loop, not letting me apply any change to the configuration. The warning does not appear either.

Hi, it seems like the error isn't caused by this commit. I changed the filter to stable branch and still encountered this error.

It seems to be a native error in Moodle, and it's resolved in versions after 402.

I found info in this thread

https://tracker.moodle.org/browse/MDL-80735?focusedId=1058774&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1058774
(I tried 403, and indeed, the error no longer occurs).

**Review by carla-at-wiris — APPROVED** [review]

**Review by jgonzalez-at-wiris — APPROVED** [review]

**Review by usantos-at-wiris — APPROVED** [review]
Good Job!
